tests/test_integrated_channels/test_sap_success_factors/test_exporters/test_content_metadata.py

```diff
@@ -423,3 +425,106 @@ def test_transform_course_description(self, course, expected_description):
                 'value': expected_description
             }
         ]
+
+    def _merge_dicts(self, dict1, dict2):
```

Comment:
this can be move in `test_utils.py` and used it from there here in SAPSF and Cornerstone.

Reply:
Done